packages/config/config/devices/0x027a/zen17.json

```diff
@@ -60,118 +60,246 @@
 				}
 			]
 		},
-		"2": {
-			"label": "Input Type for S1 Terminal",
-			"description": "Note: The device must be excluded (without resetting it) and re-included after changing to/from values 4-10",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 10,
-			"defaultValue": 1,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Momentary switch",
-					"value": 0
-				},
-				{
-					"label": "Toggle switch (follow switch)",
-					"value": 1
-				},
-				{
-					"label": "Toggle switch (change on toggle)",
-					"value": 2
-				},
-				{
-					"label": "Garage door (momentary mode)",
-					"value": 3
-				},
-				{
-					"label": "Water sensor",
-					"value": 4
-				},
-				{
-					"label": "Heat alarm",
-					"value": 5
-				},
-				{
-					"label": "Motion alert",
-					"value": 6
-				},
-				{
-					"label": "Door sensor (open/close)",
-					"value": 7
-				},
-				{
-					"label": "CO alarm",
-					"value": 8
-				},
-				{
-					"label": "CO2 alarm",
-					"value": 9
-				},
-				{
-					"label": "On/off report (dry contact switch/sensor)",
-					"value": 10
-				}
-			]
-		},
-		"3": {
-			"label": "Input Type for S2 Terminal",
-			"description": "Note: The device must be excluded (without resetting it) and re-included after changing to/from values 4-10",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 10,
-			"defaultValue": 1,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Momentary switch",
-					"value": 0
-				},
-				{
-					"label": "Toggle switch (follow switch)",
-					"value": 1
-				},
-				{
-					"label": "Toggle switch (change on toggle)",
-					"value": 2
-				},
-				{
-					"label": "Garage door (momentary mode)",
-					"value": 3
-				},
-				{
-					"label": "Water sensor",
-					"value": 4
-				},
-				{
-					"label": "Heat alarm",
-					"value": 5
-				},
-				{
-					"label": "Motion alert",
-					"value": 6
-				},
-				{
-					"label": "Door sensor (open/close)",
-					"value": 7
-				},
-				{
-					"label": "CO alarm",
-					"value": 8
-				},
-				{
-					"label": "CO2 alarm",
-					"value": 9
-				},
-				{
-					"label": "On/off report (dry contact switch/sensor)",
-					"value": 10
-				}
-			]
-		},
+		"2": [
+			{
+				"$if": "firmwareVersion >= 1.4",
+				"label": "Input Type for S1 Terminal",
+				"description": "Note: The device must be excluded (without resetting it) and re-included after changing to/from values 4-11",
```

Comment:
Should we also use >= 4 in the description here and then only have a conditional extra value? The only apparent downside is the max would need to equal 11 even for 1.3 and below.